content/en/security/cloud_security_management/guide/eBPF-free-agent.md

```diff
@@ -0,0 +1,559 @@
+        ;;
+  restart)
+        stop
+        start
+        ;;
+  *)
+        echo $"Usage: $0 {start|stop|restart}"
+        exit 1
+esac
+
+exit 0
+```
+
+#### Docker
+
+For Docker application deployments, you should modify your Dockerfile to wrap your application like this:
+
+```shell
+FROM datadog/agent-dev:7-55-1-beta-host-ebpfless-v0 AS datadogagent
+
+FROM ubuntu:latest
+
+COPY --from=datadogagent /opt/datadog-agent/embedded/bin/cws-instrumentation .
+
+ENTRYPOINT ["/cws-instrumentation", "trace", "--"]
+
+CMD ["/bin/bash", "-c", "while true; do sleep 1; echo my app is running; done"]
+```
+
+When running your docker application, it’s important to give it an additional capability by adding `--cap-add=SYS_PTRACE` to your `docker run` command.
+
+You also have to connect the container to Datadog on port 5678 by doing one of the following:
+
+- Launch both containers with the `--network` host option.
+- Use the [Docker network][6] feature to run both containers on the same bridge network.
+
+#### Kubernetes
+
+Here is an example of a values.yaml file used to deploy a wrapped application together with the Datadog eBPF-Free agent:
+
+```shell
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+ name: "my_app"
+ namespace: default
+spec:
+ replicas: 1
+ selector:
+   matchLabels:
+     app: "my_app"
+ template:
+   metadata:
+     labels:
+       app: "my_app"
+     name: "my_pod"
+   spec:
+     initContainers:
+     - name: cws-instrumentation-init
+       image: public.ecr.aws/datadog/agent-dev:7-55-1-beta-host-ebpfless-v0
+       command:
+         - "/opt/datadog-agent/embedded/bin/cws-instrumentation"
+         - "setup"
+         - "--cws-volume-mount"
+         - "/cws-instrumentation-volume"
+       volumeMounts:
+         - name: cws-instrumentation-volume
+           mountPath: "/cws-instrumentation-volume"
+       securityContext:
+         runAsUser: 0
+     containers:
+     - name: "my_app"
+       image: "my_image:latest"
+       command:
+         - "/cws-instrumentation-volume/cws-instrumentation"
+         - "trace"
+         - "--"
+         - "my_entrypoint"
+       volumeMounts:
+         - name: cws-instrumentation-volume
+           mountPath: "/cws-instrumentation-volume"
+           readOnly: true
+     - name: datadog-agent
+       image: public.ecr.aws/datadog/agent-dev:7-55-1-beta-host-ebpfless-v0
+       env:
+         - name: DD_API_KEY
+           value: "<DD_API_KEY>"
+         - name: DD_RUNTIME_SECURITY_CONFIG_ENABLED
+           value: "true"
+         - name: DD_RUNTIME_SECURITY_CONFIG_EBPFLESS_ENABLED
+           value: "true"
+     volumes:
+       - name: cws-instrumentation-volume
+     serviceAccountName: datadog-agent
+     shareProcessNamespace: true
+```
+{{% /tab %}}
+
+{{% tab "Attach mode" %}}
+The wrap mode is recommended because the attach mode has the following limitations: 
+
+- It misses all initialization made by the application until Datadog attaches to it.
+- More performance overhead.
```

Comment:
Maybe we can add the why again here: because when attaching we cannot setup any seccomp profile